connections/src/main/java/com/stripe/android/connections/ConnectionsSheetActivity.kt

```diff
@@ -30,7 +30,6 @@ internal class ConnectionsSheetActivity : AppCompatActivity() {
     @VisibleForTesting
     internal var viewModelFactory: ViewModelProvider.Factory =
         ConnectionsSheetViewModel.Factory(
-            { application },
```

Comment:
Just as an example, now the parent component provides the `Application` instance instead.